keanu-python/keanu/tensor.py

```diff
@@ -72,8 +72,11 @@ def __get_tensor_from_scalar(scalar: primitive_types) -> JavaObject:
             raise NotImplementedError("Generic types in a ndarray are not supported. Was given {}".format(type(scalar)))
 
     @staticmethod
-    def _to_ndarray(java_tensor: JavaObject) -> numpy_types:
+    def _to_ndarray(java_tensor: JavaObject, primitive: bool = False) -> Any:
```

Comment:
I find 'primitive' parameter name a little bit confusing - perhaps "return_as_primitive" would be better?